packages/E2ETest/wdio/pages/BasePage.ts

```diff
@@ -3,89 +3,44 @@
  * Licensed under the MIT License.
  */
 
-import {
-  REACT_CONTROL_ERROR_TEST_ID,
-  BACK_BUTTON,
-  TREE_DUMP_RESULT,
-} from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+import { TREE_DUMP_RESULT } from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+
+const ELEMENT_LOADED_TIMEOUT = 60000;
 
 export function By(testId: string): WebdriverIO.Element {
   return $('~' + testId);
 }
 
-export function wait(timeout: number) {
-  return new Promise(resolve => {
-    setTimeout(resolve, timeout);
-  });
-}
-
 export class BasePage {
   isElementLoaded(element: string): boolean {
     return By(element).isDisplayed();
   }
 
-  waitForElementLoaded(element: string, timeout?: number) {
+  waitForElementLoaded(element: string) {
     browser.waitUntil(
-      () => {
-        return this.isElementLoadedOrLoadBundleError(element);
-      },
-      this.timeoutForPageLoaded(timeout),
```

Comment:
Replaced this function with a constant since we only ever used the constant instead of the optional param